tests/robot/tests/2_jenkins_models/2.0_check_jenkins_models.robot

```diff
@@ -37,6 +42,8 @@ Checking property update callback
     Ensure model property has been updated  ${model_id}  ${model_version}  ${edge}  ${TOKEN}  ${MODEL_WITH_PROPS_PROP}  1
     Ensure model API call result field is correct  ${model_id}  ${model_version}  ${edge}  ${TOKEN}  ${MODEL_WITH_PROPS_ENDPOINT}  result  30   a=1  b=2
 
+    Ensure metric present  ${model_id}  ${model_version}  ${MODEL_WITH_PROPS_ENDPOINT}
```

Comment:
> 
> 
> Maybe we have to add tests for non-default endpoint? What do you think?

I have already added a case for non-default endpoint. Is that OK?

Reply:
OK, that's good